dbus/mako.c

```diff
@@ -65,6 +65,42 @@ static int handle_dismiss_last_notification(sd_bus_message *msg, void *data,
 	return sd_bus_reply_method_return(msg, "");
 }
 
+static int handle_dismiss_notification(sd_bus_message *msg, void *data,
+		sd_bus_error *ret_error) {
+	struct mako_state *state = data;
+
+	uint32_t id = 0;
+	uint32_t dismiss_group = 0;
+	int ret = sd_bus_message_read(msg, "ub", &id, &dismiss_group);
+	if (ret < 0) {
+		return ret;
+	}
+
+	if (id == 0) {
+		id = state->last_id;
+	}
+
+	if (wl_list_empty(&state->notifications)) {
```

Comment:
Nit: technically this `if` is unnecessary, `wl_list_for_each` is a no-op on empty lists.

Reply:
That's the pattern in `handle_invoke_action` and I assumed it was there for a good reason. Removed from here.